sidecar/sidecar.go

```diff
@@ -93,29 +93,6 @@ func Run(conf config.Config) error {
 
 	appSupervisor := supervisor.NewAppSupervisor(&conf)
 
-	if conf.Log {
-		//Replace the LOGSTASH_REPLACEME string in filebeat.yml with
-		//the value provided by the user
-		//TODO: Make this configurable
-		filebeatConf := "/etc/filebeat/filebeat.yml"
```

Comment:
So this is gone right? Because the diff shows that its being removed. Its needed only in examples, not testing. And if the examples work correctly, it will validate that this subsystem works properly. 

The changes to be made to examples will be in the form of adding a sidecar.yml config file, where we add a shell script that would generate a filebeat.yaml config and launch filebeat.